docker/pulsar/pom.xml

```diff
@@ -57,6 +57,29 @@
       <id>docker</id>
       <build>
         <plugins>
+          <plugin>
+            <groupId>pl.project13.maven</groupId>
+            <artifactId>git-commit-id-plugin</artifactId>
```

Comment:
Although, this plugin seems a bit wordy and it's licensed under LGPL (although we don't bundle its source code so it should not be an issue).

I can trust [mavanagaiata](https://github.com/koraktor/mavanagaiata) plugin that has been used in ZooKeeper projects for years.

Reply:
We already used this plugin in the codebase:
https://github.com/apache/pulsar/blob/43b3622cc7e7f746ca9920fdc704dc7448767ac7/pom.xml#L1928-L1930

I think we can consider replacing it with another PR.

Reply:
.SGTM